src/oic/utils/sdb.py

```diff
@@ -1091,3 +1158,28 @@ def read(self, token):
             raise WrongTokenType("Not a grant token")
 
         return self._db[key]
+
+    def get_by_sub(self, sub):
+        return self._db.get_by_sub(sub)
+
+    def get_by_uid(self, uid):
+        return self._db.get_by_uid(uid)
+
+    def get_by_smid(self, smid):
+        return self._db.get_by_smid(smid)
+
+    def get_by_sub_and_client_id(self, sid, client_id):
```

Comment:
These are sort of a retrieve and verify and should probably be in a client and not in SessionDB.

Reply:
OK